client/src/bootstrap.js

```diff
@@ -62,19 +66,27 @@ function loop() {
   dt += (t - old_time);
   old_time = t;
   songTime = mm.music.currentTime;
-  while (dt > MS_PER_FRAME) {
+  while (dt >= MS_PER_UPDATE_FRAME) {
     sm.update();
     mm.update();
     MOUSE.scrollX = 0;
     MOUSE.scrollY = 0;
     tick++;
```

Comment:
tick should be obsolete now, right?  (it's at least referenced when sending a state)